.bazelrc

```diff
@@ -24,6 +24,9 @@ build --define cxxopts=-std=c++17
 build --features=-per_object_debug_info
 # Suppress deprecated declaration warnings due to extensive transitive noise from protobuf.
 build --copt -Wno-deprecated-declarations
+# Abseil Flag literals are compiled out by default on mobile platforms. These flags are needed for
+# Envoy's runtime system. This option prevents them from being compiled out.
+build --copt -DABSL_FLAGS_STRIP_NAMES=0
```

Comment:
Should this be defined upstream instead? Fine here for now.

Reply:
Yes, I think it makes most sense to land here first so we can unblock this update, then remove it once it's available upstream.